docs/source/daml/reference/packages.rst

```diff
@@ -20,7 +20,7 @@ You can specify a different path for the Daml archive by using the ``-o`` flag:
 
   daml build -o foo.dar
 
-For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <deploy-ref_overview>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
+For details on how to upload a Daml archive to the ledger, see the :ref:`deploy documentation <././deploying>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
```

Comment:
```suggestion
For details on how to upload a Daml archive to the ledger, see the :doc:`deploy documentation </deploying>`. The rest of this page will focus on how to import a Daml package in other Daml projects.
```
:ref: references a label that you set somewhere. You can see the previous one in the deleted file via `.. _deploy-ref_overview:`. `:doc` is used to reference a specific document which you’re doing here.

This is documented in https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html